equinix/resource_fabric_cloud_router.go

```diff
@@ -99,30 +278,86 @@ func resourceCloudRouterRead(ctx context.Context, d *schema.ResourceData, meta i
 	return setCloudRouterMap(d, CloudRouter)
 }
 
+func packageCloudRouterGoToTerra(packageType *v4.CloudRouterPackageType) *schema.Set {
+	packageTypes := []*v4.CloudRouterPackageType{packageType}
+	mappedPackages := make([]interface{}, len(packageTypes))
+	for i, packageType := range packageTypes {
+		mappedPackages[i] = map[string]interface{}{
+			"code": packageType.Code,
+		}
+	}
+	packageSet := schema.NewSet(
+		schema.HashResource(&schema.Resource{Schema: resourcesFabricCloudRouterPackageSch()}),
+		mappedPackages,
+	)
+	return packageSet
+}
+
+func orderCloudRouterGoToTerra(order *v4.Order) *schema.Set {
```

Comment:
This should be moved to internal package in the schema subpackage. So it should be moved to `internal/schema/fabric_common_mapping_helpers.go`

Reply:
Along with any other methods that have common schemas. We identified these as the common ones: order, notifications, account, change log, location, and project